sc-locked-asset-factory/src/locked_asset_token_merge.rs

```diff
@@ -21,30 +21,34 @@ pub trait LockedAssetTokenMergeModule:
     + nft_deposit::NftDepositModule
     + token_merge::TokenMergeModule
 {
-    #[endpoint(mergeTokens)]
     fn merge_tokens(
```

Comment:
done

Reply:
the flow might be similar in these two cases but the functions they call differ. it can be done with function pointers or inheritance but i think that'll make the review process even harder. might refactor after things are a little more stable